### DISCUSSION TITLE
Nilai p-value utk t-test dan Z-test menghasilkan nilai p-value =1 dan Apakah yg dimaksud dg uji 2 mean pairwise ?

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#539] [Q&A] [answered]
Melanjutkan pertanyaan sebelumnya terkait p-value, berikutnya didapatkan hasil Nilai p-value = 1 utk t-test dan Z-test ;
<img width="467" alt="image" src="https://user-images.githubusercontent.com/123436963/217980934-b62a2c5d-1c11-4860-8ccf-9e535c72d9c1.png">

<img width="476" alt="image" src="https://user-images.githubusercontent.com/123436963/217981315-1988bc11-3d02-457d-967a-3fc1e62ebd46.png">

sehingga disarankan utk Uji 2 Mean pairwise krn data tsb masih pada entitas yg sama.
Terkait kondisi tsb diatas, Apakah nilai p-value =1 tsb tidak wajar dan perlu dilakukan uji 2 Mean pairwise ? Terimakasih

**Comment by ranydwi** [comment, marked as answer]
Dear @Micoalgo 

Terima kasih untuk pertanyaan yang masuk, sebelumnya saya masih belum menangkap tujuan/goal dari pengujian Bapak. Mungkin boleh dielaborasi lagi Pak untuk tujuan pengujiannya ya.

Terkait nilai p-value = 1 pada pengujian Z-test dan t-test, hal ini mungkin saja terjadi, akan tetapi sangat rare terjadi. Hal ini mungkin terjadi apabila nilai kedua buah mean/rata-rata pengujian nilainya mendekati 0. Karena nilai pengujian mean antara populasi dan sampel dalam satuan range yang besar, hal ini juga bisa saja terjadi Pak.

Semoga membantu
Regards

**Comment by Micoalgo** [comment]
Terimakasih jawabannya mba Rany,  rencananya pada sesi grup mentoring hari
ini on site akan saya elaborasi lagi utk detail pertanyaannya 🙏
